react/src/components/CustomizedImageList.tsx

```diff
@@ -101,20 +102,41 @@ const CustomizedImageList: React.FC<PropsWithChildren> = ({ children }) => {
     },
   );
 
-  const [commitForgetAndUntag, isInflightForgetAndUntag] =
-    useMutation<CustomizedImageListForgetAndUntagMutation>(graphql`
-      mutation CustomizedImageListForgetAndUntagMutation($id: String!) {
-        untag_image_from_registry(image_id: $id) {
+  const [commitForget, isInFlightForget] =
+    useMutation<CustomizedImageListForgetMutation>(graphql`
+      mutation CustomizedImageListForgetMutation($id: String!) {
+        forget_image_by_id(image_id: $id) {
           ok
           msg
         }
-        forget_image_by_id(image_id: $id) {
+      }
+    `);
+
+  const [commitUntag, isInFlightUntag] =
+    useMutation<CustomizedImageListUntagMutation>(graphql`
+      mutation CustomizedImageListUntagMutation($id: String!) {
+        untag_image_from_registry(image_id: $id) {
           ok
           msg
         }
       }
     `);
 
+  // TODO: when BA-1905 resolved.
+  // const [commitPurgeImage, isInFlightPurgeImage] =
+  //   useMutation<CustomizedImageListPurgeImageMutation>(graphql`
+  //     mutation CustomizedImageListPurgeImageMutation($id: String!) {
```

Comment:
```suggestion
  //   useMutation<CustomizedImageListPurgeMutation>(graphql`
  //     mutation CustomizedImageListPurgeMutation($id: String!) {
```
Please match naming convention with other mutations.